### PR TITLE
we are merging the freaking frontend scaffold in; it's not good rn; we have to merge master in to our branch

### PR DESCRIPTION
**Pull request by vrautela**
pls help

### CODE REVIEW

**Comment by rymaju** [comment]
oh my. thats a lot of merge conflicts

File: src/auth/axios.ts [hunk 1]
```diff
@@ -42,4 +47,48 @@ const AppAxiosInstance: AxiosInstance = axios.create({
 //   return Promise.reject(error);
 // };
```
Comment (jackblanc) [inline src/auth/axios.ts:48]: The stuff above should get deleted

File: src/containers/signup/index.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```
Comment (rymaju) [inline src/containers/signup/index.tsx:1]: I would prefer to get rid of this, but I get if we just have it and never use it

File: src/store.ts [hunk 3]
```diff
@@ -49,9 +52,30 @@ export const initialStoreState: C4CState = {
 const thunkExtraArgs: ThunkExtraArgs = {
   authClient,
   tokenService,
-  protectedApiClient,
+  // protectedApiClient
```
Comment (rymaju) [inline src/store.ts:55]: Are we removing this?

File: src/components/LinkButton.tsx [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
-import { Button } from 'antd';
 import { useHistory } from 'react-router-dom';
+import { BaseButtonProps } from 'antd/lib/button/button';
 // onClick is overwritten
 
-export const LinkButton: React.FC<any> = ({ to, ...rest }) => {
+interface LinkButtonProps extends  BaseButtonProps{
+  to: string;
+}
+
```
Comment (rymaju) [inline src/components/LinkButton.tsx:10]: spacing is weird, did we run npm run check and prettier?

**Review by rymaju — COMMENTED** [review]
Remember to run prettier and fix the errors from the Github actions, then we should be good to merge

File: src/components/navbar/index.tsx [hunk 5]
```diff
@@ -89,6 +90,10 @@ const NavBar: React.FC<NavBarProps> = ({ tokens }) => {
   // Dropdown menu options for the logged in
   const userMenu = (
     <Menu>
+      <Menu.Item>Change Primary Account Email</Menu.Item>
+      <Menu.Item>Account Details</Menu.Item>
+      <Menu.Item>Change Password</Menu.Item>
+      <Menu.Item>Deactivate Account</Menu.Item>
```
Comment (jackblanc) [inline src/components/navbar/index.tsx:96]: Is this right? Don't they all go to the same place?

**Review by jackblanc — APPROVED** [review]
One comment with menu items otherwise lgtm

**Pull request merged**